src/components/PasswordSection.jsx

```diff
@@ -1,24 +1,17 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
-import get from 'lodash/get'
 
-import { useClient } from 'cozy-client'
 import Button from 'cozy-ui/transpiled/react/Button'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 import Typography from 'cozy-ui/transpiled/react/Typography'
 
+import { useWithoutPassword } from 'hooks/useWithoutPassword'
+
 const PasswordSection = () => {
   const { t } = useI18n()
-  const client = useClient()
-
-  // If we have no capabilities, consider that we can login to the cozy with password
-  const canAuthWithPassword = get(
-    client,
-    'capabilities.can_auth_with_password',
-    true
-  )
+  const withoutPassword = useWithoutPassword()
 
-  return canAuthWithPassword ? (
+  return !withoutPassword ? (
```

Comment:
Does this mean that an OIDC user can never set a password? I think i'm missing something as the rest of the code is constructed as if it was possible.

Reply:
Answer: the settings's option "edit password" is hidden for a passwordless cozy, but the user will be able to set a password when activating the 2FA (then the password IS the 2FA)